mkdocs/docs/linux/index.md

```diff
@@ -0,0 +1,2 @@
+
+**The tutorial has been moved to [here](https://docs.hpc.ugent.be/linux-tutorial)**
```

Comment:
I would expose the URL here rather than hiding it behind `here`, And let's add a page title too:

```md
# This tutorial has moved!

See <https://docs.hpc.ugent.be/linux-tutorial> .

Please update your bookmarks!